integration-tests/cucumber-tests-platform/src/test/java/org/opensmartgridplatform/cucumber/platform/glue/steps/database/core/ScheduledTaskSteps.java

```diff
@@ -44,7 +44,7 @@
     @Autowired
     private ScheduledTaskRepository scheduledTaskRepository;
 
-    @Given("a scheduled \"([^\"]*)\" task")
+    @Given("a scheduled {string} task")
```

Comment:
The original regex matches a string enclosed in double-quotes. The new Cucumber expression does not take into account the double-quotes.

I can't find any feature files using this step.

Reply:
Most changes were based on failing tests, but near the end I did some regex based searching and found one (I think) more potential regex issue, and replaced it.
I have intentionally been a bit careful about extra changes beside the upgrade, but if you think this had better been removed, I can take it out.